main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
The terms say that the game must create 10 stories; for this, we need to change the code.
```py
while (loop  >=  10):
```